InstallAPIcalls.py

```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+import requests
+import numpy as np
+import time
+import pandas as pd
+import json
+
+def UtilityCheck(utility):
+    valid_utilities = ["speculoos_power"]
+    error_string = "The utility provided is not available within Bayou Energy."
+    if utility not in valid_utilities:
+        return error_string
+    else:
+        return None
+    
+def BayouAPICustomer(utility, email):
+    # Bayou has two environments, staging, as shown below, and production, bayou_domain="bayou.energy"
+
+    bayou_domain = "staging.bayou.energy"
+
+    # Get and manage API keys at f"https://{bayou_domain}/dashboard/keys"
+
+    # API reference: https://docs.bayou.energy/v2.0/reference/authentication 
+    
+    # Note: This is mine (Chad's) personal API key. We may want to hide this during production.
+
+    bayou_api_key = "test_221_f6f1a2d9daef91272215d4761f52ff80465c7dee839b4585a0aab96ddc284159"
```

Comment:
I hid this using Python's dotenv module, let me know if this still works on your side!